.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+<!-- 
+Things to check before submitting a pull request:
+  * Label this PR with the correct semver label.
```

Comment:
I don't think third parties can do that actually.

Reply:
Makes sense, I'll add "if you can"